discovery/pkg/openstack/reconciler.go

```diff
@@ -116,6 +118,24 @@ func (r *Reconciler) reconcile() {
 		log.Errorf("error listing OpenStack projects: %v", err)
 		return
 	}
+
+	// import white list
+	tmp := projects[:0]
+	if whitelist, ok := os.LookupEnv("PROJECT_WHITELIST"); ok && len(whitelist) > 0 {
```

Comment:
I think this check should be performed in main.go, and then the list of projects flowed/injected into a field on the `Reconciler` type. 

That will make it more testable.

Reply:
Thank you for your reply.
I change this imports env variable "WATCHED_PROJECTS", which is renamed, from main.go.